### PR TITLE
Introduce Nth node.

### PR DESCRIPTION
**Pull request by ProgVal**
Introduced in https://github.com/ProjetPP/Documentation/pull/59

### CODE REVIEW

File: ppp_datamodel/nodes/list_operators.py [hunk 1]
```diff
@@ -99,19 +100,43 @@ class Or(ListOperator):
...
+def First(list_):
+    logger.warning('Using deprecated alias `First(L)`. '
+                   'Use `first(L)` or `Nth(1, L)` instead.')
+    return first(list_)
+def Last(list_):
+    logger.warning('Using deprecated alias `Last(L)`. '
+                   'Use `last(L)` or `Nth(-1, L)` instead.')
+    return last(list_)
+def first(list_):
+    return Nth(1, list_)
```
Comment (Tpt) [inline ppp_datamodel/nodes/list_operators.py]: Nth starts from 0 and not 1

Comment (ProgVal) [inline ppp_datamodel/nodes/list_operators.py]: fixed

**Comment by ProgVal** [comment]
Note: the Travis build fails only because this branch does not have Python 2 compatibility.

**Comment by Ezibenroc** [comment]
Ok for me.

**Pull request merged**